syntax = "proto3";

package syft.core.node.common.service;

import "proto/core/common/common_object.proto";
import "proto/core/io/address.proto";

message SimpleMessage {
  syft.core.common.UID msg_id = 1;
  syft.core.io.Address address = 2;
  syft.core.io.Address reply_to = 3;
  syft.core.common.UID id_at_location = 4;
}

message SimpleReplyMessage {
  syft.core.common.UID msg_id = 1;
  syft.core.io.Address address = 2;
  string repr = 3;
}